### PR TITLE
adds new tutorial

### PR DESCRIPTION
**Pull request by lwinfree**
...for workshop training at TU Delft

Part of issue https://github.com/frictionlessdata/project/issues/718

### CODE REVIEW

**Comment by lwinfree** [comment]
Hi @shashigharti! I've added a new tutorial to this folder :-) I thought I would tag you to review it so you can practice that? If it looks OK, please go ahead and merge. Thanks!

**Comment by shashigharti** [comment]
Thank you! looks good to me!. 

Few comments:
- It would be nice to add "open in colab" button link. 
[Open In Colab](https://colab.research.google.com/github/frictionlessdata/frictionless-py/blob/5d13a6db2757ca1db4a25693d7ead7087d50c7c0/docs/tutorials/notebooks/frictionless-RDM-workflows.ipynb#scrollTo=SmB2Y4_rfUUv)
- a typo: 'machine readible'

**Comment by lwinfree** [comment]
thanks @shashigharti! I just made those changes :-)

**Comment by lwinfree** [comment]
actually I think I added the badge incorrectly... let me try again

**Comment by lwinfree** [comment]
ok @shashigharti it should be all fixed now! :-)

**Comment by shashigharti** [comment]
Thank you!
